src/blocks/product-card.css

```diff
@@ -0,0 +1,125 @@
+:root {
+  --color-aquamarine: #00acdc;
+  --color-dark-white: #f3f3f3;
+  --color-navy-black: #060b35;
+  --color-light-brown: #616070;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  background: white;
+  border: 1px solid var(--color-dark-white);
+  border-radius: 5px;
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  padding: 0 16px;
+  box-sizing: border-box;
+}
+
+.product-card__image {
+  padding: 0 3px;
+  width: 160px;
+  height: 134px;
+  padding-top: 32px;
+  margin-bottom: 40px;
+}
+
+.product-card__name {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-navy-black);
+  height: 36px;
+  margin-bottom: 4px;
+  margin-top: 0;
+}
+
+.product-card__code {
+  width: 99px;
+  height: 14px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-light-brown);
+  margin-bottom: 16px;
+  margin-top: 0;
+}
+
+.product-card__review {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  height: 16px;
+  margin-bottom: 24px;
+}
+
+.product-card__review-count {
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  width: 53px;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  width: 166px;
+  height: 18px;
+  margin-bottom: 16px;
+}
+
+.product-card__price-label {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  display: flex;
+  align-items: flex-end;
+  color: var(--color-light-brown);
+}
+
+.product-card__price-value {
+  width: 50px;
+  height: 18px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  display: flex;
+  align-items: flex-end;
+  color: var(--color-navy-black);
+}
+
+.product-card__buy {
+  width: 164px;
```

Comment:
Usually setting width is a bad practice as it will make any responsive styling really hard. Avoid setting width directly on an element. This button is taking the whole width of the card (without paddings) so there is no need of setting it. Imagine if you will need to make changes to this card. For example change it width then you will need to readjust all other elements so it is very hard work.

Reply:
I deleted this `width`.